great_expectations/rule_based_profiler/profiler.py

```diff
@@ -43,10 +43,11 @@ def __init__(
             profiler_config: Variables and Rules configuration as a dictionary
             data_context: DataContext object that defines a full runtime environment (data access, etc.)
         """
+        self._profiler_config = profiler_config
         self._data_context = data_context
         self._rules = []
 
-        rules_configs: Dict[str, Dict] = profiler_config.get("rules", {})
+        rules_configs: Dict[str, Dict] = self._profiler_config.get("rules", {})
```

Comment:
This has no impact but now that `self._profiler` is a class attribute, I utilized it instead of the `profiler` arg passed to the constructor wherever possible.